### PR TITLE
Improve .gitignore

### PR DESCRIPTION
**Pull request by se-bastiaan**
Ignore more files that should not be tracked by git.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,17 +1,51 @@
...
-.idea
+# Java class files
+*.class
+
+# Generated files
+bin/
+gen/
+.idea/
 *.iml
-*.ipr
```
Comment (brianwernick) [inline .gitignore]: We should keep the *.ipr and *.iws for those that want to use that format for AndroidStudio projects.

File: .gitignore [hunk 2]
```diff
@@ -1,17 +1,51 @@
-/build
-library/build
-demo/build
```
Comment (brianwernick) [inline .gitignore]: would you please keep the library/build and demo/build exclusions to be safe, even though the current files are accounted for wit the .class, bin/, gen/ .apk, etc.

Comment (se-bastiaan) [inline .gitignore]: done

**Comment by brianwernick** [comment]
Thanks :+1:

**Pull request merged**